File: java/prototype/Brochure.java
package prototype;

public class Brochure extends MarketingMaterial {
    public Brochure(String layout, String content, String color) {
        super(layout, content, color);
    }

    @Override
    public Brochure cloneMarketingMaterial() {
        return new Brochure(this.getLayout(), this.getContent(), this.getColor());
    }

}

File: C#/Prototype/Brochure.cs
namespace Prototype;

public sealed class Brochure : MarketingMaterial
{
    public Brochure(string layout, string content, string color) : base(layout, content, color)
    {
    }

    public override MarketingMaterial CloneMarketingMaterial()
        => new Brochure(Layout, Content, Color);
}
